recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler:                    # [win]
+- vs2017                       # [win]
+cxx_compiler:                  # [win]
+- vs2017                       # [win]
```

Comment:
Trying downgrading to VS 2017. Not ideal as the default is VS 2019. However if we need it, this is an option

Reply:
Looks like we may not have VS 2017 on the Azure images any more. So that doesn't look like an option. Will revert back to the defaults